packages/tool-cache/src/tool-cache.ts

```diff
@@ -5,7 +5,7 @@ import * as os from 'os'
 import * as path from 'path'
 import * as httpm from '@actions/http-client'
 import * as semver from 'semver'
-import * as uuidV4 from 'uuid/v4'
+import uuidV4 from 'uuid/v4'
```

Comment:
reaction due to moving `"esModuleInterop": true` to the root `tsconfig.json` - which previously was only defined in `packages/github/tsconfig.json`